sdbc_test.go

```diff
@@ -101,7 +105,7 @@ func prepareDatabase(
 	tb.Helper()
 
 	req := testcontainers.ContainerRequest{
-		Name:  "sdbc_" + toSlug(tb.Name()),
+		// Name:  "sdbc_" + toSlug(tb.Name()),
```

Comment:
To keep the code as clean as possible we should not keep commented code, but rather remove it completely. The git history (and GitHub) will provide old code versions if needed later :)
```suggestion
```